### ISSUE TITLE
Scripthash Unsubscribe support

### ISSUE THREAD

**Issue by JoleMile** [BlueWallet/rn-electrum-client#6]
Does this library provide the support for `blockchain.scripthash.unsubscribe` documented here https://electrumx.readthedocs.io/en/latest/protocol-methods.html#blockchain-scripthash-unsubscribe?

If not, would it be hard to add?

**Comment by Overtorment** [comment]
no, but should be trivial to add. see https://github.com/BlueWallet/rn-electrum-client/blob/99ebcc649d91a8dc39bea7964b02dd9ead464aa4/index.js#L134
